### DISCUSSION TITLE
Is image available for arm64 / how to build or pull it?

### DISCUSSION THREAD

**Discussion by ludobegins** [kartoza/docker-geoserver#554] [Q&A]
HI, I tried:
```
docker pull kartoza/geoserver:2.23.0 --platform linux/arm64
```
But i get `WARNING: image with reference kartoza/geoserver was found but does not match the specified platform: wanted linux/arm64, actual: linux/amd64`

So my question is: Is the image available for arm64? If yes, how to pull it or build it?
I saw arm64 mentioned in "Notes for MacOS M1 Chip".

The reason is I'd like to build the image in an EC2 instance that has this architecture.

Edit: i saw that issue #436 addresses that, will try to work with that info. But any tip on running image on aws graviton is appreciated.

Thanks

**Comment by NyakudyaA** [comment]
This is available on docker hub
